build/Dockerfile

```diff
@@ -63,6 +63,18 @@ RUN set -x \
 	&& mkdir -p $(dirname $CLUSTER_PATH) \
 	&& chown $STEAM_USER:$STEAM_USER $(dirname $CLUSTER_PATH)
 
+ARG MODS
```

Comment:
The `ARG/ENV` lines should be moved to the existing `ARG/ENV` block.

Reply:
Same reason. I don't want to download the game again only for a change in the mods.